libs/entry-components/styles/modules/vendors/redactor/_buttons-generator.scss

```diff
@@ -0,0 +1,22 @@
+@use 'sass:map';
+
+$material-button-background: #237BBB;
```

Comment:
As agreed, if we don’t add specific values in our custom theme, it should inherit Angular Material's default values. However, this depends on the Redactor configuration on project level because there are two ways of adding it. If Redactor is passed within generated Formly components, it will have a Material container wrapper and will inherit values by default; otherwise, if we create custom redactor configuration, it will not inherit values so I added default values so now, both cases are covered.